app/src/main/res/values/strings.xml

```diff
@@ -10,7 +10,7 @@
     <string name="something_went_wrong">Something went wrong!</string>
     <string name="edit_button">Edit button</string>
     <string name="submit">Submit</string>
-    <string name="add_item_to_list">Add item to list</string>
+    <string name="add_item_to_list">Add item</string>
```

Comment:
It has to be the separate string `add_item`.
The reason is that `add_item_to_list` is used on a different screen (add_fragment).

Reply:
Ok. I've also created the string at your translate file. Although, since I don't this language, I would be very appreciated if you could completed this for me.